src/auth/credential-provider.ts

```diff
@@ -1,6 +1,6 @@
 import {decodeJwt} from '../utils/jwt';
 
-export interface ICredentialProvider {
+export interface CredentialProvider {
```

Comment:
Thank you for freeing us from our [Hungarian](https://en.wikipedia.org/wiki/Hungarian_notation) oppressors.